python/test/unit/fem/test_complex_assembler.py

```diff
@@ -8,14 +8,11 @@
 import ufl
 import dolfin
 import numpy as np
-import pytest
 from ufl import dx, grad, inner
-
-# FIXME: Move to dolfin_utils.test.skips
-pytestmark = pytest.mark.skipif(not dolfin.config.has_petsc_complex,
-                                reason="Only works in complex mode.")
+from dolfin_utils.test.skips import skip_if_not_complex
```

Comment:
What about  `pytestmark = skip_if_not_complex` at the beginning after the imports instead a decorator for each function.
I think it would be clearer that all tests should be skipped if we are not in complex mode.

Reply:
Ok, in the current case it does not make much difference because we have only two tests.

Reply:
Honestly I originally removed it (because it was a variable not used and broke the tests) because I didn't realize that the import has a side effect. I find it weird, but I am fine with doing it your way.